Packs/DeveloperTools/pack_metadata.json

```diff
@@ -32,6 +32,10 @@
         "CommonTypes": {
             "mandatory": true,
             "display_name": "Common Types"
+        },
+        "GetServerURL": {
```

Comment:
remove this please.
GetServerUrl is moving to CommonScripts so no need to add it here

Reply:
done